Automattic/jetpack
# AI Assistant: Remove page content from prompt

Fixes #31446

## Proposed changes:

* Removes the post content entirely, leaving only the relevant content

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add an AI Assistant block
* Try the block options. The results should keep working as normal
* Check the logs. The page content should not be included
